src/OpenLoco/S5/S5.cpp

```diff
@@ -397,7 +397,7 @@ namespace OpenLoco::S5
         {
             // Shift data after companies to correct location
             auto src = reinterpret_cast<uint8_t*>(&state) + 0x49EA24;
-            auto dst = src + 0x1C20;
+            auto dst = src + 0x1C20; // std::size(Company::cargoUnitsDistanceHistory) * std::size(state.companies)
```

Comment:
Any particular reason why this has been commented out?

Reply:
As that is not how i want it done but it will do for now.